### PR TITLE
Feat/button component v2

### PR DESCRIPTION
**Pull request by youssef-night**

### CODE REVIEW

File: libs/ui/components/inputs/button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    size,
+    backgroundColor,
+    hoverBackgroundColor,
+    border,
+    borderRadius,
+    handleOnClick,
+    text,
+    color,
+    w,
+    h,
```
Comment (m-Nightly) [inline libs/ui/components/inputs/button/index.tsx:15]: Not being uised? I would use the Chakra Props and spread

**Review by m-Nightly — CHANGES_REQUESTED** [review]

**Comment by m-Nightly** [comment]
Good job !

File: libs/ui/components/inputs/button/index.tsx (hunk 1 above)
Comment (youssef-night) [reply libs/ui/components/inputs/button/index.tsx:15]: This pull request is only for ButtonIcon, so this other component will be make later

**Review by m-Nightly — APPROVED** [review]
LGTM !

**Pull request merged**